lms/djangoapps/save_for_later/management/commands/send_course_reminder_emails.py

```diff
@@ -0,0 +1,88 @@
+"""
+Management command to send reminder emails.
+"""
+
+import logging
+
+from textwrap import dedent
+from datetime import datetime, timedelta
+
+from django.conf import settings
+from django.core.management import BaseCommand
+from django.contrib.auth.models import User  # lint-amnesty, pylint: disable=imported-auth-user
+
+from lms.djangoapps.save_for_later.helper import send_email
+from lms.djangoapps.save_for_later.models import SavedCourse
+from common.djangoapps.student.models import CourseEnrollment
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from common.djangoapps.util.query import use_read_replica_if_available
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+    """
+    Command to send reminder emails to those users who
+    saved course by email but not register within 15 days.
+
+
+    Examples:
+
+        ./manage.py lms send_course_reminder_emails --batch-size=100
+    """
+    help = dedent(__doc__)
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--batch-size',
+            type=int,
+            default=1000,
+            help='Maximum number of users to send reminder email in one chunk')
+
+    def handle(self, *args, **options):
+        """
+        Handle the send save for later reminder emails.
+        """
+
+        reminder_email_threshold_date = datetime.now() - timedelta(
+            days=settings.SAVE_FOR_LATER_REMINDER_EMAIL_THRESHOLD)
+        saved_courses_ids = SavedCourse.objects.\
+            filter(reminder_email_sent=False, modified__lt=reminder_email_threshold_date).values_list('id', flat=True)
```

Comment:
It's better read in this way:
```suggestion
        saved_courses_ids = SavedCourse.objects.filter(
            reminder_email_sent=False, modified__lt=reminder_email_threshold_date
        ).values_list('id', flat=True)
```